.babelrc

```diff
@@ -1,4 +1,4 @@
 {
     "presets": ["@babel/preset-env"],
-    "plugins": ["@babel/transform-runtime"]
+    "plugins": ["@babel/transform-runtime","@babel/plugin-proposal-class-properties"]
```

Comment:
@ishimwebernard for what purpose are you using "@babel/plugin-proposal-class-properties" module?